Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -1023,7 +996,7 @@ public static bool Is8Dot3(string name)
 
         public static bool Is8Dot3Char(char ch)
         {
-            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%�-{}()@'`&".IndexOf(ch) != -1;
+            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%£-{}()@'`&".IndexOf(ch) != -1;
```

Comment:
I noticed the mail part too - didn't think of it.
The file is UTF-8 now, and the sign is correctly set?

Would suck to suddenly skip a certain character because the source file used a different symbol for it ... :)